pkg/openscap/openscap.go

```diff
@@ -109,11 +109,18 @@ func (s *defaultOSCAPScanner) getInputCVE(dist int) (string, error) {
 	return cveFileName, err
 }
 
+func strOrDefault(s string, d string) string {
```

Comment:
looks fine to me, I really wish there was a ternary operator in golang for things like this (never thought I'd be wishing for that).  Only suggestion is to move to a util package and make it public so we don't rewrite it again.